feature/staff/src/main/java/io/github/droidkaigi/confsched2020/staff/ui/StaffsFragment.kt

```diff
@@ -68,14 +72,27 @@ class StaffsFragment : Fragment() {
 
         val groupAdapter = GroupAdapter<ViewHolder<*>>()
         binding.staffRecycler.adapter = groupAdapter
+        binding.staffRecycler.itemAnimator = object : DefaultItemAnimator() {
+            override fun animateAdd(holder: RecyclerView.ViewHolder?): Boolean {
+                dispatchAddFinished(holder)
```

Comment:
I have a question 🙋 
Why this is needed? Can you add a comment?

Reply:
In "Motion" repository, editor comments `We animate item additions on our side, so disable it in RecyclerView.`.

ref <https://github.com/android/animation-samples/blob/232709094f9c60e0ead9cf4873e0c1549a9a8505/Motion/app/src/main/java/com/example/android/motion/demo/stagger/StaggerActivity.kt#L61>

In my opinion, that comment is based on `animationAdd` method's documents.

> Called when an item is added to the RecyclerView. Implementors can choose whether and how to animate that change, but must always call dispatchAddFinished(RecyclerView.ViewHolder) when done, either immediately (if no animation will occur) or after the animation actually finishes. 

ref <https://developer.android.com/reference/androidx/recyclerview/widget/SimpleItemAnimator.html#animateAdd(androidx.recyclerview.widget.RecyclerView.ViewHolder)>

In this case, we set custom animation (Stagger transition), so we should call `dispatchAddFinished`( and `dispatchAddStarting` for running animation).

---

I plan to add following comments.
Does this comment look good?

```
// Because custom RecyclerView's animation, set custom SimpleItemAnimator implementation.
//
// see https://developer.android.com/reference/androidx/recyclerview/widget/SimpleItemAnimator.html#animateAdd(androidx.recyclerview.widget.RecyclerView.ViewHolder)
// see https://github.com/android/animation-samples/blob/232709094f9c60e0ead9cf4873e0c1549a9a8505/Motion/app/src/main/java/com/example/android/motion/demo/stagger/StaggerActivity.kt#L61
 binding.staffRecycler.itemAnimator = ~~
```

Reply:
Plus one point, `DefaultItemAnimator` 's `animateAdd` is following code.

```java
    @Override
    public boolean animateAdd(final RecyclerView.ViewHolder holder) {
        resetAnimation(holder);
        holder.itemView.setAlpha(0);
        mPendingAdditions.add(holder);
        return true;
    }
```

Probably, if you do not set the SimpleItemAnimator implementation, the custom animation will be cleared.
(Sorry, not tried yet...)

Reply:
Thanks! Looks good!

Reply:
:airplane: 34f0d8f